src/content/docs/apm/apm-ui-pages/monitoring/find-performance-problems-apm-transactions-page.mdx

```diff
@@ -0,0 +1,374 @@
+---
+title: 'Find performance problems using the APM Transactions page'
+tags:
+  - APM
+  - APM UI pages
+  - Monitoring
+translate:
+  - jp
+metaDescription: APM's Transactions page shows app, browser, and other transaction details, average response time, and throughput.
+redirects:
+  - /docs/applications-menu/web-transactions
+  - /docs/applications-menu/background-tasks
+  - /docs/applications-menu/transactions-dashboard
+  - /docs/apm/applications-menu/monitoring/transactions-dashboard
+  - /docs/apm/applications-menu/monitoring/transactions-page
+  - /docs/apm/applications-menu/monitoring/transactions-page-find-performance-problems
+  - /docs/apm/applications-menu/monitoring/transactions-page-find-specific-performance-problems
+  - /docs/apm/transactions/intro-transactions/transactions-page-find-specific-performance-problems
+  - /docs/apm/apm-ui-pages/monitoring/transactions-page-find-specific-performance-problems
+---
+
+The APM **Transactions** page helps you identify transactions to look at more closely in order to troubleshoot performance problems and avoid errors. 
+
+In general, the longer a given transaction takes, and the more time it takes compared to other transactions, the more likely it is that it's a source of issues for a given app. Similarly, low throughput can be an indicator that not everything is working as it should. 
+
+We recommend you make a habit of reviewing app performance data on the APM **Transactions** page more closely in order to scan for new issues and diagnose the cause of problems you're already aware of.
```

Comment:
Maybe something here about using the page to "_learn_ how your service performs"? That learning idea comes up a lot when people try to define observability.